dorm/dorm/Presenter/SplitView/CustomSplitViewController.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  CustomSplitViewController.swift
+//  dorm
+//
+//  Created by JongHo Park on 2022/07/20.
+//
+
+import Combine
+import UIKit
+
+class CustomSplitViewController: UISplitViewController {
```

Comment:
```suggestion
final class CustomSplitViewController: UISplitViewController {
```

더이상 상속되지않는다는거 확실하게 명시하기 위해서 final이 들어가는 것이 좋을 것 같아요 ~